addons/account/account_move_line.py

```diff
@@ -274,7 +274,7 @@ def _default_get(self, cr, uid, fields, context=None):
                     #if there exist some unbalanced accounting entries that match the journal and the period,
                     #we propose to continue the same move by copying the ref, the name, the partner...
                     move = move_obj.browse(cr, uid, move_id, context=context)
-                    data.setdefault('name', move.line_id[-1].name)
+                    data.setdefault('name', move.line_id[-1].name if len(move.line_id) > 0 else 'Empty Movement')
```

Comment:
aprt that I don't really see the added value of such a patch, you forgot the _() to enable translations